ArtU/ArtU/Views/GalleryView.swift

```diff
@@ -0,0 +1,179 @@
+//
+//  GalleryView.swift
+//  ArtU
+//
+//  Created by Eliza Hales on 11/26/22.
+//
+
+import SwiftUI
+
+struct TransitionIsActiveKey: EnvironmentKey {
```

Comment:
Would it be best to put these in a ViewController or helper file? This way we could keep the view 'dumb'. What do you think?